rosdep/python.yaml

```diff
@@ -6311,6 +6311,29 @@ python3-helyos_agent_sdk-pip:
   '*':
     pip:
       packages: [helyos_agent_sdk]
+python3-hid:
```

Comment:
pip-only keys should have a `-pip` suffix: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-1